Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -79,6 +79,10 @@ internal class CustomerIOImplementation: CustomerIOInstance {
         identifier: String,
         body: RequestBody
     ) {
+        if identifier.isEmpty {
```

Comment:
also should we check for both both blank and empty?

Reply:
What is difference between blank and empty ? 🤔

Reply:
let identifier = "   " // a blank string

the `identifier.isEmpty` above will return false, but i dont think we should even allow this

Reply:
✅